Formula/tcpreplay.rb

```diff
@@ -12,23 +12,19 @@ class Tcpreplay < Formula
     sha256 "ab0379428462fbdc2653feae2bcc404cf6b6d2129ddf0461019c53794ce87e4f" => :mavericks
   end
 
-  depends_on "libdnet" => :recommended
+  depends_on :xcode
```

Comment:
It's always effectively been assumed by the formula because `"--with-libpcap=#{MacOS.sdk_path}/usr"` crashes if you don't have Xcode.